source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -0,0 +1,169 @@
+#include "extensions/filters/http/aws_lambda/aws_lambda_filter.h"
+
+#include <string>
+#include <vector>
+
+#include "envoy/upstream/upstream.h"
+
+#include "common/common/fmt.h"
+#include "common/common/hex.h"
+#include "common/crypto/utility.h"
+#include "common/http/headers.h"
+#include "common/protobuf/utility.h"
+
+#include "extensions/filters/http/well_known_names.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AwsLambdaFilter {
+
+namespace {
```

Comment:
just wanted to clarify that this is conventional in Envoy? Im more used to seeing Util classes etc.

Reply:
We've talked about that in a previous PR :)
I create util classes when the function is using in more than on TU.